src/Makefile.OCaml

```diff
@@ -364,6 +365,8 @@ ifeq ($(OSARCH), OpenBSD)
     OCAMLDOT=true
   endif
 else
+  # \todo POSIX does not specify a -t option to type, or the presence
+  # of bash.
   ifeq ($(shell echo type -t ocamldot | bash), file)
```

Comment:
I see two possible ways here. First, get rid of everything related to ocamldot? Alternatively, remove all the logic for setting `OCAMLDOT=true` and replace the `ifdef OCAMLDOT` below with `ifneq ($(shell command -v ocamldot),)` (which is the POSIX way).

Reply:
Typed in wrong box earlier :-(    My intent is to merge this with the \todo, thus leaving cleaning up the ocamldot stanza as future work.  This PR is intentionally limited to comments and removing content that isn't used.